packages/core/admin/_internal/node/createBuildContext.ts

```diff
@@ -218,7 +218,11 @@ const filterPluginsByAdminEntry = (plugin: Plugin) => {
 
     return isLocalPluginWithLegacyAdminFile;
   } catch (err) {
-    if (isError(err) && 'code' in err && err.code === 'MODULE_NOT_FOUND') {
+    if (
+      isError(err) &&
+      'code' in err &&
+      (err.code === 'MODULE_NOT_FOUND' || err.code === 'ERR_PACKAGE_PATH_NOT_EXPORTED')
```

Comment:
why was this change needed?

Reply:
I made this change, because the we get an error about `strapi-admin` missing from the exports package, which realistically we can ignore because not all plugins have an admin part :)